README.md

```diff
@@ -1 +1,66 @@
-# meals-palace
\ No newline at end of file
+## Meals-Palace
+
+> Project Description.
+The meals-palace provides a web application displaying a list of foods along with their inttegrated recipes. It is based on MealsDM and Involvement APIs.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript -Webpack -JS Modules
+- Git
+- Jest(for testing)
+- Webpack
+
+## Live Demo 
+
+[Live Demo Link](https://youtu.be/3n6VZYeCeAs)
```

Comment:
- [ ] Since this link leads to your video presentation, I recommend changing its name from `Live demo link` to `video presentation`.